openmdao/docs/features/recording/reading_data.rst

```diff
@@ -22,6 +33,22 @@ from the first driver iteration.
 .. note::
     Note that you can use either the promoted or absolute names when accessing variables.
 
+Getting Variable Data from Case Recording of a Problem
+------------------------------------------------------
+
+Here are the methods to use when retrieving data from the recording of a `Problem` case recorder.
```

Comment:
I think these methods should also be valid for any type of recorder.. inputs and outputs are pulled from the appropriate system based on the `recording_requester`